<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoDoIt</string>
  </property>
  <property name="windowIcon">
   <iconset resource="qrc.qrc">
    <normaloff>:/image/Attachment.png</normaloff>:/image/Attachment.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QListView" name="listView">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_editTask"/>
   <addaction name="separator"/>
   <addaction name="action_addTask"/>
   <addaction name="action_removeTask"/>
   <addaction name="separator"/>
   <addaction name="action_startTask"/>
   <addaction name="action_stopTask"/>
   <addaction name="separator"/>
   <addaction name="action_doTask"/>
   <addaction name="action_debug"/>
  </widget>
  <action name="action_addTask">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/add.png</normaloff>:/image/add.png</iconset>
   </property>
   <property name="text">
    <string>创建任务</string>
   </property>
   <property name="toolTip">
    <string>创建任务</string>
   </property>
  </action>
  <action name="action_removeTask">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/delete.png</normaloff>:/image/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除任务</string>
   </property>
   <property name="toolTip">
    <string>删除任务</string>
   </property>
  </action>
  <action name="action_startTask">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/run.png</normaloff>:/image/run.png</iconset>
   </property>
   <property name="text">
    <string>开启任务</string>
   </property>
   <property name="toolTip">
    <string>开启任务</string>
   </property>
  </action>
  <action name="action_editTask">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/edit.png</normaloff>:/image/edit.png</iconset>
   </property>
   <property name="text">
    <string>编辑任务</string>
   </property>
   <property name="toolTip">
    <string>编辑任务</string>
   </property>
  </action>
  <action name="action_setting">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/setting.png</normaloff>:/image/setting.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="toolTip">
    <string>设置</string>
   </property>
  </action>
  <action name="action_debug">
   <property name="text">
    <string>测试</string>
   </property>
   <property name="toolTip">
    <string>测试</string>
   </property>
  </action>
  <action name="action_stopTask">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/stop.png</normaloff>:/image/stop.png</iconset>
   </property>
   <property name="text">
    <string>停止任务</string>
   </property>
   <property name="toolTip">
    <string>停止任务</string>
   </property>
  </action>
  <action name="action_stopAllTask">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/stop.png</normaloff>:/image/stop.png</iconset>
   </property>
   <property name="text">
    <string>停止所有任务</string>
   </property>
   <property name="toolTip">
    <string>停止所有任务</string>
   </property>
  </action>
  <action name="action_startAllTask">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/chenked.png</normaloff>:/image/chenked.png</iconset>
   </property>
   <property name="text">
    <string>开启所有任务</string>
   </property>
   <property name="toolTip">
    <string>开启所有任务</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
   <property name="iconText">
    <string>关于AutoDoIt</string>
   </property>
   <property name="toolTip">
    <string>关于AutoDoIt</string>
   </property>
  </action>
  <action name="action_autoStart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>开机启动</string>
   </property>
   <property name="toolTip">
    <string>开机启动</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="action_doTask">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/do.png</normaloff>:/image/do.png</iconset>
   </property>
   <property name="text">
    <string>手动触发</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections/>
</ui>
